src/components/elements/off-canvas-menu.js

```diff
@@ -24,7 +24,7 @@ import DTrader from 'images/svg/dtrader-icon.svg'
 import Forex from 'images/svg/forex-nav.svg'
 import Help from 'images/svg/menu/help-center.svg'
 import Leadership from 'images/svg/menu/leadership.svg'
-import MarginTrading from 'images/svg/margin-trading-nav.svg'
+import CFD from 'images/svg/margin-trading-nav.svg'
```

Comment:
The image name and Variable name is not similar.

Reply:
same as here...